backend/app/alembic/versions/add_prompt_management_tables.py

```
@@ -0,0 +1,99 @@
+"""添加提示词管理相关表
+
+Revision ID: 0001
+Revises: 
+Create Date: 2023-01-01 00:00:00.000000
+
+"""
+from alembic import op
+import sqlalchemy as sa
+import sqlmodel
```

Ruff (0.11.9)

`sqlmodel` imported but unused

Remove unused import: `sqlmodel`

(F401)